0200-mapping.Rmd

```diff
@@ -0,0 +1,158 @@
+
+- Make sure to sync the project [following these instructions](https://merginmaps.com/docs/manage/synchronisation/#how-to-synchronise-changes-in-mergin-maps) after making any edits.
+
+
+### Using Mergin on a phone
+
+- We find that Mergin works much better (faster rendering) with *airplane* mode on.    
+
+### Field Data Entry into Mergin
+
+- Over the last year we have seen uncommon issues with syncing and data corruption that provide even more incentive to be sure to do paper copies of the forms as well. 
+- There are links to the methods docs in the forms with the intention that you can save them to your phone.  Obviously, the provincial method documents are key.
+- Switching between themes is very useful in the field. You can add more in QGIS.
+- Site ID is important:
+  - If a site has a PSCIS ID then that is used as the identifier. 
+  - If there is no PSCIS ID we use the "Crossing - Modelled ID" (a.k.a "My Crossing Reference") as the "Modelled ID". If there is neither (rare but usually occurs when a stream is not mapped) we use a numeric ID based on the date (YYYYMMDD01 - example 2023070502 - would be our second assessment for today).
+  - If there is an associated chris_culvert_id ("MoTi" identifier) we include that as well. We take a photo of the road with the PSCIS ID or Modelled ID in the corner and upload photos to mergin that are
+- When we first arrive at a site we take a photo of the road with the PSCIS ID or Modelled ID in the corner. This allows easy reference to which photos are for which sites. When we are done filling out the paper form we take a photo of it (for data backup and to make it easy to see which photos are from where).
+- At this point in time the climate change metrics are a touch subjective and poorly defined.  We will be refining them later but for now it is effective for comparing one site to the next. Keen on your feedback as this was passed on to us as is and we have some ideas too about ways to simplify/standardize things.
+- The "FDS" and "Site Card/Fish form" stuff in the "ignore_mobile/templates" directory are what we use for Habitat Confirmation (Phase 2) work as it allows us to standardize data collection and upload more detailed habitat/fish information to the province.  
+
+
+### Office
+
+- Project creation is done with `bcdata.sh` and `background_layers.sh` found in https://github.com/NewGraphEnvironment/dff-2022/tree/master/scripts/qgis.  See the readme for instructions. 
+
+- There is a directory in the project called `ignore_mobile` which contain the site photos. The project directory also contains the methods and templates for paper data collection, as well as the excel files that are used to upload the data to the province. The provincial method documents are obviously key to review, print and have on hand in the field.
```

Comment:
Info about `ignore_mobile` can be tucked somewhere as a detail.  The project directory also contains the methods and templates for paper data collection is important but can go where the mergin info specific to qgis is located as at this point i think those pdfs can only be accessed from there.  there is a link to the methods in the fieldforms though so that can be noted as it can be opened in the field if there is cell and airplan isn't turned off :>

Reply:
moved